pkg/requesthandler/blocks.go

```diff
@@ -15,7 +15,7 @@ import (
 func (r *RequestHandler) BlockByID(blockID iotago.BlockID) (*iotago.Block, error) {
 	block, exists := r.protocol.Engines.Main.Get().Block(blockID)
 	if !exists {
-		return nil, ierrors.Wrapf(echo.ErrNotFound, "block not found: %s", blockID.ToHex())
+		return nil, ierrors.WithMessagef(echo.ErrNotFound, "block %s not found", blockID.ToHex())
```

Comment:
I notice we sometimes use blockID.ToHex() and other times just blockID. Which is preferred?

Reply:
`%s` with just `blockID` is preferred because block ID implements aliases, so we want to call the `String()` method on the ID (which is what `%s` will result in) and not `ToHex()`. That way we get the alias printed in error messages when we have an alias set (such as within tests). Thanks for pointing that out!